### PR TITLE
ensure that (non-array) floats are tested as input during forward pass

### PR DESCRIPTION
**Pull request by rsokl**

### CODE REVIEW

File: tests/wrappers/uber.py [hunk 1]
```diff
@@ -119,6 +120,9 @@ def __init__(
             A callable that is fed the generated arrays and keyword arguments that will
             be fed to ``mygrad_func``. If ``assumptions`` returns ``False``, that test
             case will be marked as skipped by hypothesis.
+
+        permit_0d_array_as_float : bool, optional (default=True)
+            If True, drawn 0D arrays will potentially be case to numpy-floats.
```
Comment (davidmascharka) [inline tests/wrappers/uber.py]:
```suggestion
            If True, drawn 0D arrays will potentially be cast to numpy-floats.
```

File: tests/wrappers/uber.py [hunk 2]
```diff
@@ -243,6 +248,16 @@ def wrapper(shapes: hnp.BroadcastableShapes, constant, data: st.DataObject):
                 for value in self.index_to_no_go.get(i, ()):
                     assume(np.all(arr != value))
 
+            if self.permit_0d_array_as_float:
+                # potentially cast a 0D array as a float
+                arrs = tuple(
+                    arr.item()
+                    if arr.ndim == 0
+                    and data.draw(st.booleans(), label="arr-{} to float".format(n))
```
Comment (davidmascharka) [inline tests/wrappers/uber.py]: is there a plan to drop 3.5?

Comment (rsokl) [reply tests/wrappers/uber.py]: Not until it officially hits EOL :/

**Comment by rsokl** [comment]
@davidmascharka should be all set now

**Pull request merged**